Cargo.toml

```diff
@@ -15,44 +15,34 @@ path = "src/lib.rs"
 [dependencies]
 clap = "^2.33"
 derive_more = "0.15"
-eth2_hashing = "0.3.0"
-eth2_key_derivation = { git = "https://github.com/ChorusOne/lighthouse", rev = "ef90a15aae791435a4180d7e6bc1d8e29ced6e2e"}
-eth2_keystore = { git = "https://github.com/ChorusOne/lighthouse", rev = "ef90a15aae791435a4180d7e6bc1d8e29ced6e2e"}
-eth2_network_config = { git = "https://github.com/ChorusOne/lighthouse", rev = "ef90a15aae791435a4180d7e6bc1d8e29ced6e2e" }
-eth2_ssz = "0.4.1"
-eth2_wallet = { git = "https://github.com/ChorusOne/lighthouse", rev = "ef90a15aae791435a4180d7e6bc1d8e29ced6e2e"}
+ethereum_hashing = "1.0.0-beta.2"
+eth2_key_derivation = { git = "https://github.com/ChorusOne/lighthouse", rev = "ed9ebf47ce743ea6afc8c8fe7d82e14da5b4d26c"}
```

Comment:
should we also sync our fork (in a separate issue)?

Reply:
The fork have been updated too, see https://github.com/ChorusOne/lighthouse/tree/chorus-patches